force-app/main/default/lwc/crmNavTaskQuickText/crmNavTaskQuickText.js-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>61.0</apiVersion>
+    <isExposed>true</isExposed>
+    <masterLabel>NKS NAV Task Quick Text</masterLabel>
```

Comment:
Kan vi kalle den noe annet som f.eks. NKS NAV Task Communal Qucik Text, siden den bare skal brukes for kommunale henvendelser?